Split/Common/Utils/Logger.swift

```diff
@@ -7,6 +7,53 @@
 
 import Foundation
 
+struct TimeChecker {
+
+    private static var startTime: Int64 = 0
+    private static let tag = "[SPTPRF] "
+    private static let showTimestamp = true
+    private static let showSinceMsg = true
+    static func start() {
+        startTime = Date.nowMillis()
+        Logger.v("\(tag) TimeChecker started at: \(startTime)")
+    }
+
+    static func logInterval(_ msg: String) {
+        let now = Date.nowMillis()
+        let interval = now - startTime
+        Logger.v("\(tag) \(msg) \(formatTimestamp(now)) \(formatIntervalSinceStart(interval))")
+    }
+
+    static func logTime(_ msg: String) {
+        Logger.v("\(tag) \(msg) \(formatIntervalSinceStart(Date.nowMillis()))")
+    }
+
+    static func logInterval(_ msg: String, startTime: Int64) {
+        Logger.v("\(tag) \(msg) \(Date.nowMillis() - startTime) ms \(formatTimestamp(Date.nowMillis()))")
+    }
+
+    static func formatInterval(_ interval: Int64) -> String {
```

Comment:
Is the `formatInterval` method used? Maybe it can be removed, because it seems to do the same as `formatIntervalSinceStart`